joplin/db/system-generated/seeding.datadump.json

```diff
@@ -9331,8 +9211,8 @@
 {
   "model": "users.user",
   "fields": {
-    "password": "pbkdf2_sha256$150000$wg2Rdshfdqly$3VTEDmmvBrEigN93vPEgx/dMD8675aHOMAtCGPv/vWw=",
-    "last_login": "2019-09-27T05:10:50.496Z",
+    "password": "pbkdf2_sha256$100000$9fpvtivuWOYg$rOB3vDHoYJyEYs35zNdIJ0BmHvR8n5daAVwz7HoeC2Q=",

```

Comment:
Did this password not get sanitized? Or was this added locally?

Reply:
I honestly am not sure what's going on with it, but I've been able to log in as multiple users with sanitized pws so there's that at least